college_scorecard/college_scorecard.rb

```diff
@@ -0,0 +1,31 @@
+require 'csv'
+
+file = CSV.open "./2013_college_scorecards.csv", headers: true, header_converters: :symbol
+value = ARGV[1]
+opt_arg = ARGV[2].to_i if ARGV[2]
+
+if ARGV.first == "by_state"
```

Comment:
This was a good stab at this challenge in Ruby. Let's put this in a class next time!